### DISCUSSION TITLE
Adding description to "IframeBlocked" cookie

### DISCUSSION THREAD

**Discussion by jasmindworschak** [dargmuesli/nuxt-cookie-control#206] [Q&A] [answered]
Hello :)

When using isIframeBlocked option, is there an intended way of adding a description to "functional cookies" under "optional cookies" ?

I initially tried adding a description to the const blockIframes in the module.mjs which works, but is of course impractical and hacky.

Currently, I'm setting the description like so:
`moduleOptions.cookies.optional[0].description = 'I want to have a description here :)'`

<img width="543" alt="image" src="https://github.com/dargmuesli/nuxt-cookie-control/assets/72686558/2f96798d-a1b6-41c4-b5c4-eea7cf5043ed">

This seems better, but still not ideal.

Thank you in advance for your help!

**Comment by dargmuesli** [comment, marked as answer]
Right now there is no set way to specify that description. Ideally, for that to be changed, the whole "functional cookie" logic should be implemented in a way that allows for that cookie to be configurable as any other cookie.
